Common/src/Stream/Worker/TaskRequestExtensions.cs

```diff
@@ -40,27 +41,27 @@ namespace ArmoniK.Core.Common.Stream.Worker;
 [PublicAPI]
 public static class TaskRequestExtensions
 {
-  public static IEnumerable<ProcessReply.Types.CreateLargeTaskRequest> ToRequestStream(this IEnumerable<TaskRequest> taskRequests,
-                                                                                       TaskOptions?                  taskOptions,
-                                                                                       int                           chunkMaxSize)
+  public static IEnumerable<CreateTaskRequest> ToRequestStream(this IEnumerable<TaskRequest> taskRequests,
+                                                               TaskOptions?                  taskOptions,
+                                                               int                           chunkMaxSize)
   {
     var fsm = new ProcessReplyCreateLargeTaskStateMachine(NullLogger.Instance);
     fsm.InitRequest();
     if (taskOptions is not null)
     {
-      yield return new ProcessReply.Types.CreateLargeTaskRequest
+      yield return new CreateTaskRequest
                    {
-                     InitRequest = new ProcessReply.Types.CreateLargeTaskRequest.Types.InitRequest
+                     InitRequest = new CreateTaskRequest.Types.InitRequest
                                    {
```

Comment:
Stupid question: {}and () are equivalent in c#?

Reply:
It depends on what you want to do.
Usually, {} is a kind of rapid initialization of the internal data whereas () is the constructor.

Reply:
As a first approximation, 
```
var t = new T
{
  Toto = XXX;
};
```
is the same as:
```
var t = new T();
t.Toto = XXX;
```

For more details: https://docs.microsoft.com/en-us/cpp/cpp/initializing-classes-and-structs-without-constructors-cpp?view=msvc-170